python/dataproc_templates/redshift/README.md

```diff
@@ -0,0 +1,139 @@
+# Prerequisites
+
+## Required JAR files
+
+These templates requires the jar file to be available in the Dataproc cluster.
+User has to download the required jar file and host it inside a GCS Bucket, so that it could be referred during the execution of code.
+
+* spark-redshift.jar
+* spark-avro.jar
+* RedshiftJDBC.jar
+* minimal-json.jar
+
+Once the jar file gets downloaded, please upload the file into a GCS Bucket and export the below variable
+
+```
+export JARS=<comma-seperated-gcs-bucket-location-containing-jar-file> 
+```
+
+## Redshift JDBC URL syntax
+
+* Redshift
+```
+jdbc:redshift://[Redshift Endpoint]:[PORT]/<dbname>?user=<username>&password=<password>
+```
+
+
+## Other important properties
+
+* Required Arguments
+
+    * Temp S3 Location
+    ```
+    redshifttogcs.s3.tempdir="s3a://[Bucket-Name]/[tempdir-path]" 
+    ```
+    * Redshift IAM Role ARN
+    ```
+    redshifttogcs.iam.rolearn="arn:aws:iam::Account:role/RoleName"
+    ```
+    * S3 Access Key
+    ```
+    redshifttogcs.s3.accesskey="xxxxxxxx"
+    ```
+    * S3 Secret Key
+    ```
+    redshifttogcs.s3.secretkey="xxxxxxxx"
+    ```
+
+* You can either specify the source table name or have SQL query within double quotes. Example,
+
+```
+redshifttogcs.input.table="employees"
+```
+
+# Redshift To GCS
```

Comment:
take title and description to top of the readme

Reply:
modified as suggested